src/testdir/test_termcodes.vim

```diff
@@ -220,10 +221,23 @@ func Test_1xterm_mouse_wheel()
     call MouseWheelUp(1, 1)
     call assert_equal(1, line('w0'), msg)
     call assert_equal([0, 7, 1, 0], getpos('.'), msg)
+
+    call MouseWheelRight(1, 1)
```

Comment:
I wanted to test the window column position too here, like the asserts above for mouse wheel up/down, but I didn't find any function to report the current window column position. Is there any?